.github/workflows/build.yml

```diff
@@ -157,10 +148,7 @@ jobs:
         uses: ChrisCarini/intellij-platform-plugin-verifier-action@v0.0.2
         with:
           plugin-location: plugin-artifact/*.zip
-          ide-versions: |
-            ideaIC:2019.3
-            ideaIC:2020.1
-            ideaIC:2020.2
+          ide-versions: ideaIC:2020.3
```

Comment:
I reduced the number of verifications that are run on a build.
I don't think we gain a lot when checking against a lot of IntelliJ-Versions.